tox.ini

```diff
@@ -15,12 +15,13 @@ deps =
    coverage
    isort
    mccabe
+   pytest
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m unittest discover -s {envsitepackagesdir}/pylint/test/ -p {posargs:*test_*}.py
+    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/
```

Comment:
@rogalski test selection is being controlled from [pytest.ini](https://github.com/PyCQA/pylint/pull/1199/files#diff-513c9b6861fc981fb2c708db477f542c)